doc/operator-creation.md

```diff
@@ -128,7 +128,20 @@ There are a few things to know and (try to) understand while developing operator
    the shared underlying subscription.
    - When you `subscribe` to an Observable, the functions or Observer you passed are used to create the final `destination`
    `Subscriber` for the chain. It's this `Subscriber` that is really also the shared `Subscriptoin` for the operator chain.
-   
+
+### Submitting a PR for an Operator
+
+Please complete these steps for each new operator added to RxJS as a pull request:
+
+- Add the operator to either Core or KitchenSink
+- It should have a `-spec.ts` tests file covering the canonical corner cases, with marble diagram tests
+- It should have a `asDiagram` test case too, if possible
+- It Should have a type definition test at the end of the spec to verify type definition for various use cases
```

Comment:
@kwonoj can you help explain what a Type Definition test is and where to find a good example?